guides/hugo-starter-guide/editor-configuration.mdx

```diff
@@ -19,39 +19,40 @@ explicit_canonical: /documentation/guides/universal-starter-guide/editor-configu
+    It is easy to supplement your Markdown content with React components, building an interactive page editing experience for your team — all without touching the underlying code. Snippets are structured components which you can create and edit to suit your needs. You can create your own custom Snippets with our <a href="https://cloudcannon.com/documentation/articles/snippets-using-mdx-components/#custom-mdx-components" target="_blank" rel="noreferrer noopener">Custom MDX Components guide</a>, but for now let’s import the default MDX Snippets configurations.
 
     Currently, you cannot configure Snippets through the Configuration GUI, however, it is easy to do manually. Navigate to your CloudCannon global configuration file through the Files tab in your navigation sidebar. The configuration file will be in the root of your directory.
 
-    CloudCannon supports all of <a target="_blank" rel="noopener noreferrer" href="https://gohugo.io/content-management/shortcodes/#use-hugos-built-in-shortcodes">Hugo’s predefined shortcodes</a>.
+    CloudCannon supports all of <a href="https://gohugo.io/content-management/shortcodes/#use-hugos-built-in-shortcodes" target="_blank" rel="noreferrer noopener">Hugo’s predefined shortcodes</a>.
 
     Your `_snippets_imports` should look something like this:
 
     <comp.CodeBlock language="YAML" source="cloudcannon.config.yml">
     ```
-    # Exclude all builtin Hugo shortcodes:
-    _snippets_imports:
-      hugo: false
+            # Exclude all builtin Hugo shortcodes:
+            _snippets_imports:
+              hugo: false
 
-    # Exclude all shortcodes by default,
-    # but import the youtube and vimeo shortcodes specifically:
-    _snippets_imports:
-      hugo:
-        include:
-          - hugo_youtube
-          - hugo_vimeo
+            # Exclude all shortcodes by default,
+            # but import the youtube and vimeo shortcodes specifically:
+            _snippets_imports:
+              hugo:
+                include:
+                  - hugo_youtube
+                  - hugo_vimeo
 
-    # Import all shortcodes by default,
-    # but exclude the embed shortcode:
-    _snippets_imports:
-      hugo:
-        exclude:
-          - hugo_embed
+            # Import all shortcodes by default,
+            # but exclude the embed shortcode:
+            _snippets_imports:
+              hugo:
+                exclude:
+                  - hugo_embed
     ```
```

Comment:
It won't let me do a suggestion :( But this shouldn't be indented — the code is indented on the page:
![Screenshot 2024-03-07 at 10 21 41 AM](https://github.com/CloudCannon/platform-documentation/assets/40188355/3e0df890-6d41-4d0c-8938-fc9dd5e0c1a4)